TSPL.docc/LanguageGuide/ControlFlow.md

```diff
@@ -759,6 +759,126 @@ if temperatureInFahrenheit <= 32 {
 Because the temperature is neither too cold nor too warm to trigger the `if` or `else if` conditions,
```

Comment:
@chuckdude Are you trying to avoid "nor" in general, or just this particular usage?  It shows up 6 other places in TSPL, so we could do a fix throughout if needed.

Reply:
neither/nor is a construct I prefer to avoid because it's a double negative. Using "or" as the conjunction should suffice.

Reply:
Thanks. Let's dig into this separately — [Chicago](https://www.chicagomanualofstyle.org/book/ed17/part2/ch05/psec234.html) lists either/or and neither/nor as correlated conjunctions. I wouldn't really characterize this construct as a double negative, but as one of the few (maybe the only?) place in English where negative concord remains.